packages/js/src/core/fetch-token.test.ts

```diff
@@ -61,7 +61,7 @@ describe('fetch access token by providing valid refresh token', () => {
 
     const tokenResponse = await fetchTokenByRefreshToken(
       {
-        clientId: 'client_id_value',
```

Comment:
we can keep core client id since it directly contacts oidc?

Reply:
OK, reverted all the changes made in js core, keeping only those in `browser` and above layers